cmake/FindTreesitter.cmake

```diff
@@ -1,4 +1,5 @@
 find_path2(TREESITTER_INCLUDE_DIR tree_sitter/api.h)
+find_path2(TREESITTER_INCLUDE_DIR tree_sitter/parser.h)
```

Comment:
Is this necessary? Looks like tests pass without it

Reply:
Not if parser.h is in the same directory as api.h. "find_path" just means "find directory containing this file"

Reply:
Also I'm pretty sure it doesn't do anything. You can't use the dame variable twice because the secind find_path won't do anything if the first path was found already